cmd/derper/derper.go

```diff
@@ -122,6 +123,8 @@ func main() {
 	letsEncrypt := tsweb.IsProd443(*addr)
 
 	s := derp.NewServer(key.Private(cfg.PrivateKey), log.Printf)
+	// TODO feature flag this? Or is it ok to be insta-stable?
```

Comment:
it's fine. can remove the comment.

Also, add the `Updates #nnn` line to the commit message.